packages/suite/src/views/settings/SettingsGeneral/AutoStart.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+import { Switch } from '@trezor/components';
+
+import { SettingsSectionItem } from 'src/components/settings';
+import { ActionColumn, TextColumn, Translation } from 'src/components/suite';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { setAutoStart } from 'src/actions/suite/suiteActions';
+
+const PositionedSwitch = styled.div`
+    align-self: center;
+`;
+
+export const AutoStart = () => {
```

Comment:
maybe lets add one commit that moves it to debug section and that can be easily reverted later. This way we can postpone asking for permission to put it here 😄